src/utils/helpers/fragment.js

```diff
@@ -236,6 +236,32 @@ const studentFragment = /* GraphQL */ `
     }
 `;
 
+const timetableFragment = /* GraphQL */ `
+    fragment TimetableWithDetails on Timetable {
+        id
+        weekday
+        type
+        date
+        starttime
+        endtime
+        subject {
+            id
```

Comment:
The following should be in the events response
{
    id: 0,
    title: 'This could be subject or event name',
    start: UTC string,
    end: UTC string,
    resourceId: id of Class where it will happen,
  }

Reply:
What about events like lunch that will happen across. How do you intend to handle the resourceId?

Reply:
Can we have such events repopulated for each class?

Reply:
Since we are not interested in the location where it will happen. The interest is to show it in the timetable for this class